circle.yml

```diff
@@ -112,7 +112,7 @@ dependencies:
     - echo BUILD="$(cat build.txt)"
     - ls -al ~/mne_data;
     - if [[ $(cat build.txt) == "html_dev" ]] || [[ $(cat build.txt) == "html_stable" ]]; then
-        python -c "import mne; mne.datasets._download_all_example_data()";
+        SUBJECTS_DIR=/home/ubuntu/mne_data/MNE-sample-data/subjects python -c "import mne; mne.datasets._download_all_example_data()";
```

Comment:
This is how you set env vars for a particular call in bash. In this case we need SUBJECTS_DIR to be set, but really only for the call to the data downloading.